CONTRIBUTING.md

```diff
@@ -66,7 +72,7 @@ and follow the [installation instructions](https://www.jetbrains.com/help/pychar
 
 ##### Download and install VS Code
 
-Downlaod and install the latest version of [VS Code](https://code.visualstudio.com/download)
+Download and install the latest version of [VS Code](https://code.visualstudio.com/download)
```

Comment:
The setup instructions are a bit confusing because they mix and match setup for PyCharm and VS Code. I would suggest that we don't include anything about IDE or setup here. The suggestions here are optional, basic, and anyone familiar with python development will already know how to set these things up. cc @sfc-gh-yixie

Reply:
I think it is good to have some instruction for the setup for people does not familiar with python. However, the description here does seems confusing to people with a mix of both VSCode and PyCharm, we can probably clear this in a different PR